### PR TITLE
Revenant Update II

### LINKED ISSUE #9092: Revenant Explosions
Revenants have a lives system which is supposed to give them some second chances if they mess up, but they inherit their ex_act from simple_animal, meaning they get gibbed regardless of stored essence or lives if they're blown up.  It doesn't make sense that a revnant would be damaged by bombs in the first place, but they should at least have the benefit of their remaining lives if they would be killed in an explosion.

### PR DESCRIPTION
**Pull request by ghost**
- Fixes revenants being killed by explosions
- Adds a Create Antagonist button for revenants
- Fixes grammar errors in the Create Antagonist admin feedback
- Adds sounds and chat notifications to Create Antagonist alerts for candidates
- Removed Mind Blast
- Mind Spike now has a cooldown of 4 seconds (up from 1.5)
- Harvest now takes 8 seconds instead of 5 and the revenant must be adjacent to the target
- Hypnotize now adds 30 seconds of sleep (up from 12)
- Revenants now start with 3 strikes (up from 2)
- Fixes revenants dying with strikes
- Admins are messaged when a revenant has no mind, and then afterwards if it still has no mind
- Fixes target list appearing when you are unlocking an ability
- Fixes #9092

### CODE REVIEW

**Comment by palpatine213** [comment]
It doesn't look like you're actually giving the unlock hypnotize spell to revenants at all. The unlock hypnotize spell isn't in the spells they get upon spawning.

**Comment by Xhuis** [comment]
Whoops.

File: code/modules/mob/living/simple_animal/revenant/revenant_abilities.dm [hunk 1]
```diff
@@ -50,6 +50,13 @@
 				if(5 to INFINITY)
 					user << "<span class='info'>Such a feast! [target] will yield much essence to you.</span>"
 			sleep(30)
+			if(!in_range(user, target))
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/revenant/revenant_abilities.dm]: Might want to check this; if you move and break the range, I believe this makes it so you can't absorb anyone else, permanently as it thinks you're still absorbing.

**Comment by Xhuis** [comment]
@tgstation/commit-access

**Comment by hornygranny** [comment]
closed for mergebegging

**Comment by Fox-McCloud** [comment]
Really.

**Comment by Xhuis** [comment]
man i really appreciate how coders are trying to further the progress of our server

**Comment by optimumtact** [comment]
I assumed HG was only joking

**Comment by WJohn** [comment]
Pls reopen @hornygranny 

Closing for merge begging is silly, come on.

**Comment by hornygranny** [comment]
it was bait all along

**Comment by Xhuis** [comment]
i took it

**Comment by Miauw62** [comment]
![http://knowyourmeme.com/photos/604074-bait-this-is-bait]

**Comment by Miauw62** [comment]
i fucking hate markdown

**Pull request merged**